cmd/ocm-support/sync-cloud-resources/cmd.go

```diff
@@ -0,0 +1,209 @@
+package sync_cloud_resources
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:          "sync-cloud-resources [branch-name] [csv-path]",
+	Short:        "Syncs cloud resources in AMS and generates quota rules for them",
+	Long:         "Syncs cloud resources in AMS and generates quota rules for them",
+	RunE:         syncCloudResources,
+	SilenceUsage: true,
+	Args:         cobra.ExactArgs(2),
+}
+
+var args struct {
+	dryRun bool
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(&args.dryRun, "dry-run", true, "When true, don't actually take any actions, just print the actions that would be taken")
```

Comment:
This message is quite generic, can we improve that/make more clear?

Reply:
Changed